specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/orchestratorInstances.json

```diff
@@ -0,0 +1,540 @@
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      }
+    },
+    "OrchestratorInstances": {
+      "description": "An array of OrchestratorInstance resources.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OrchestratorInstance"
+          },
+          "description": "An array of OrchestratorInstance resources."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "OrchestratorInstance": {
+      "description": "Represents an instance of a orchestrator.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "readOnly": true,
+          "$ref": "#/definitions/OrchestratorResourceProperties",
+          "description": "Properties of the provision operation request."
+        }
+      }
+    },
+    "OrchestratorResourceProperties": {
+      "description": "Properties of orchestrator",
+      "type": "object",
+      "properties": {
+        "resourceGuid": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource guid."
+        },
+        "state": {
+          "type": "string",
+          "enum": [
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Provisioning"
+          ],
+          "x-ms-enum": {
+            "name": "orchestratorInstance state",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "The current state of orchestratorInstance resource."
+        },
+        "orchestratorInstanceDetails": {
+          "description": "orchestrator details",
+          "$ref": "#/definitions/OrchestratorInstanceDetails"
+        },
+        "controllerDetails": {
+          "description": "controller details",
+          "$ref": "#/definitions/ControllerDetails"
+        }
+      }
+    },
+    "OrchestratorInstanceDetails": {
+      "description": "Properties of orchestrator",
+      "type": "object",
+      "properties": {
+        "orchestratorType": {
+          "description": "AAD ID used with apiserver",
+          "type": "string"
+        },
+        "properties": {
+          "description": "orchestrator instance properties",
+          "$ref": "#/definitions/OrchestratorInstanceProperties"
+        }
+      }
+    },
+    "OrchestratorInstanceProperties": {
+      "description": "controller details",
+      "type": "object",
+      "properties": {
+        "orchestratorAppID": {
+          "description": "AAD ID used with apiserver",
+          "type": "string"
+        },
+        "orchestratorTenantID": {
+          "description": "TenantID of server App ID",
+          "type": "string"
+        },
+        "clusterRootCA": {
```

Comment:
in what format?  Is this a secret?

Reply:
this is public root CA certificate..not secret

Reply:
In what format?  The description should make it clear what is expected here.  I.e. is it a base64 encoded public key, a name, etc...

Reply:
sure will update it..its base64 encoded